assets/js/components/notifications/WebDataStreamNotAvailableNotification.js

```diff
@@ -50,43 +51,45 @@ export default function WebDataStreamNotAvailableNotification() {
 		return null;
 	}
 
-	if ( measurementID === undefined ) {
-		// Wrap in the googlesitekit-publisher-win class to ensure the output is treated in the same way as BannerNotification,
-		// with only one instance visible on the screen at a time.
-		return (
-			<div className="googlesitekit-publisher-win">
-				<Grid>
-					<Row>
-						<Cell size={ 12 }>
-							<ProgressBar />
-						</Cell>
-					</Row>
-				</Grid>
-			</div>
-		);
-	}
-
 	return (
-		<BannerNotification
-			id="web-data-stream-not-available"
-			title={ __(
-				'Your Analytics configuration has changed',
-				'google-site-kit'
-			) }
-			description={ sprintf(
-				/* translators: 1: Google Analytics Measurement ID. */
-				__(
-					'The previously selected web data stream with measurement ID %1$s is no longer available. Please select a new web data stream to continue collecting data with Google Analytics.',
+		<Notification className="googlesitekit-publisher-win googlesitekit-publisher-win--win-error">
+			<SimpleNotification
+				title={ __(
+					'Your Analytics configuration has changed',
 					'google-site-kit'
-				),
-				measurementID
-			) }
-			ctaLink={ `${ settingsURL }#connected-services/analytics-4/edit` }
-			ctaLabel={ __( 'Update Analytics settings', 'google-site-kit' ) }
-			dismiss={ __( 'Maybe later', 'google-site-kit' ) }
-			// This is arbitrarily set to 55 minutes to ensure that the notification
-			// will become ready to be displayed again in an hour.
-			dismissExpires={ MINUTE_IN_SECONDS * 55 }
-		/>
+				) }
+				description={
+					<Description
+						text={ sprintf(
+							/* translators: 1: Google Analytics Measurement ID. */
+							__(
+								'The previously selected web data stream with measurement ID %1$s is no longer available. Please select a new web data stream to continue collecting data with Google Analytics.',
+								'google-site-kit'
+							),
+							measurementID
+						) }
+					/>
+				}
+				ctaLink={ `${ settingsURL }#connected-services/analytics-4/edit` }
```

Comment:
We don't need this prop as the `ActionsCTALinkDismiss` component below should handle everything now.